<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramDialog</class>
 <widget class="QDialog" name="HistogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="cbxRead">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Read and process tf-idf file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbxWrite">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Store results</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnView">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>View histograms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
